Epsilon.Host.Frontend/src/components/TopNavigation.vue

```diff
@@ -1,85 +1,104 @@
 <template>
-	<div class="header">
-		<img alt="logo" class="header-logo" src="../assets/logo.png" />
+	<div class="top-navigation">
+		<img alt="logo" class="top-navigation-logo" src="../assets/logo.png" />
 		<Row>
-			<Col :cols="9">
-				<!--TODO Component is used to selected users for upcoming feature-->
-				<!--                <div class="d-flex">-->
-				<!--                    <SearchBox-->
-				<!--                        v-model="store.state.currentUser"-->
-				<!--                        :items="store.state.users"-->
-				<!--                        :limit="5"-->
-				<!--                        placeholder="Student" />-->
-				<!--                </div>-->
+			<Col :cols="6">
+				<SearchBox
```

Comment:
Atm this is a not working feature thus please comment it out.

Reply:
It is a planned feature for release right, because it is necessary for teachers. Therefore I see no reason in not keeping it.